Marlin/Configuration.h

```diff
@@ -884,6 +884,15 @@
  * nozzle system or a piezo-electric smart effector.
  */
 //#define NOZZLE_AS_PROBE
+#if ENABLED(NOZZLE_AS_PROBE)
+  // Require a minimum temperature when using the nozzle as a probe. Useful on machines such as the Lulzbot series
+  // which uses a conductive nozzle or the Creality CR6 and other strain gauge sensors to prevent misreads from filament debris.
+  #define PROBE_REQUIRES_MINTEMP
+  #if ENABLED(PROBE_REQUIRES_MINTEMP)
+    #define PROBE_REQUIRES_MINTEMP_NOZZLE 150
+    #define PROBE_REQUIRES_MINTEMP_BED 50
+  #endif
```

Comment:
Is there any reason to have `PROBE_REQUIRES_MINTEMP`? It seems you could just have the other two options on their own, and commented out by default.

On a related note, is there a particular reason this should be limited to only NOZZLE_AS_PROBE? Hardened filament on my nozzle was certainly interfering with BLTOUCH probing for me a couple days ago.

Reply:
The comment should indicate that probing will automatically turn on heaters as well.

Reply:
I am not opposed to it being universal... But for a bltouch thats a heckuva filament bugger!!!! I can add to the comment a bit.

I also have no issue making them standalone instead of inside the nested name. Just the first bit of organization that popped into my head.

Reply:
> for a bltouch thats a heckuva filament bugger

It was a pretty good ooze from a 1mm nozzle. It makes a pretty sturdy column to press the bed down with!